build.sh

```diff
@@ -87,6 +87,9 @@ cargo install --target-dir prjoxide-build \
 cmake -B nextpnr-bba-build -S nextpnr-src/bba
 cmake --build nextpnr-bba-build
 
+python -m venv apycula-prefix
```

Comment:
The `python` binary should be overridable, Debian still has Python 2 as `python`.
```suggestion
${PYTHON:-python} -m venv apycula-prefix
```